src/modules/app/components/Header/Header.tsx

```diff
@@ -137,20 +140,27 @@ export class Header extends React.Component<Props, State> {
                             { user ?
                                 menuLinks
                                     .filter(link => !link.needsAuthorization || (user.isAdmin && user.isApproved))
-                                    .map(link => 
-                                            <NavLink
-                                                exact
-                                                key={link.text}
-                                                to={link.href}
-                                                className={classes ? classes.navLinkDesktop : null}
-                                                activeClassName={classes ? classes.desktopSelected : null}
+                                    .map(link =>
+                                        <NavLink
+                                            exact
+                                            key={link.text}
+                                            to={link.href}
+                                            className={classes ? classes.navLinkDesktop : null}
+                                            activeClassName={classes ? classes.desktopSelected : null}
+                                        >
+                                            <Button
```

Comment:
Remove the button, it's not good for UX, nav links are fine and is what industry leaders use for navigations

Reply:
I did not like just the text. Again, this is a question for the staff. It is hard to see the text and is not clear that they are clickable.

Reply:
this practice is used industry wide and is backed by research: 
https://getbootstrap.com/
https://github.com/
https://www.uber.com/
https://material.io/develop/web/components/top-app-bar/
https://www.nngroup.com/articles/homepage-links/ (This one doesn't mention it specifically, but the research shows good examples of nav bars that use the same practice we are)

I could scour the internet for more examples, but the bottom line is that buttons in the nav bar don't fit well. It also contradicts best practices with Material. In short, the reason is because buttons are clunky and draw attention. The navigation elements on a website are supposed to be background elements.

Reply:
Also this is one where I can talk with the staff, but they don't have experience building UI's. Even if they want something different, they aren't making an educated decision because they don't study this stuff. If need be, I can talk with them to give us more of the implementation decisions of UX

Reply:
I can agree with you on this one. I understand they are supposed to be background elements. However, I still feel like the way in which we had it before was not clean or clear. I do still think this needs to be changed, but I do agree that my solution was not the best.